### PR TITLE
[WEBDEV-822] Refactor SI and PNSI index pages

### PR DESCRIPTION
**Pull request by katylouise**
* Created ListDescriptionHelper.
* Created LaidThingListComponentsFactory.
* Updated tests and fixtures - in particular for sanitization as behaviour has changed with the latest update of the gem.

### CODE REVIEW

File: app/controllers/proposed_negative_statutory_instruments_controller.rb [hunk 1]
```diff
@@ -10,12 +10,7 @@ class ProposedNegativeStatutoryInstrumentsController < ApplicationController
   def index
     @proposed_negative_statutory_instruments = FilterHelper.filter(@api_request, 'ProposedNegativeStatutoryInstrumentPaper')
 
-    list_components = @proposed_negative_statutory_instruments.map do |proposed_negative_statutory_instrument|
-      CardFactory.new(
-        heading_text: proposed_negative_statutory_instrument.try(:proposedNegativeStatutoryInstrumentPaperName),
-        heading_url:  proposed_negative_statutory_instrument_path(proposed_negative_statutory_instrument.graph_id)
-      ).build_card
-    end
+    list_components = LaidThingListComponentsFactory.new(statutory_instruments: @proposed_negative_statutory_instruments, type: :proposed_negative_statutory_instrument).build_components
```
Comment (mattrayner) [inline app/controllers/proposed_negative_statutory_instruments_controller.rb]: Could this be done using a class method, removing the need to call `build_components`?

File: config/locales/en.yml [hunk 2]
```diff
@@ -37,6 +37,8 @@ en:
   time:
     formats:
       default: '%-d %B %Y'
+      datetime_format: '%Y-%m-%d'
```
Comment (mattrayner) [inline config/locales/en.yml]: The format should be without the hyphens

File: app/services/laid_thing_list_components_factory.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
+class LaidThingListComponentsFactory
```
Comment (mattrayner) [inline app/services/laid_thing_list_components_factory.rb]: See the above comment

File: config/locales/en.yml (hunk 2 above)
Comment (katylouise) [reply config/locales/en.yml]: Are you sure? The sample html from Peter was: "<time datetime='2017-06-13'>13 June 2017</time>".

File: app/controllers/proposed_negative_statutory_instruments_controller.rb (hunk 1 above)
Comment (katylouise) [reply app/controllers/proposed_negative_statutory_instruments_controller.rb]: Ok, I'll have a look at this.

Comment (katylouise) [reply app/controllers/proposed_negative_statutory_instruments_controller.rb]: When I came to testing it, I was thinking about this.

File: config/locales/en.yml (hunk 2 above)
Comment (mattrayner) [reply config/locales/en.yml]: Sorry, you're right - that was my mis-reading a while ago

**Pull request merged**